ga4gh/cli.py

```diff
@@ -1953,6 +1990,11 @@ def getParser(cls):
             "file must be bgzipped and indexed.")
         cls.addNameOption(addReferenceSetParser, objectType)
         cls.addDescriptionOption(addReferenceSetParser)
+        cls.addNcbiTaxonIdOption(addReferenceSetParser)
```

Comment:
These aren't shared by any other parsers, so might as well just put them in directly without the class methods. Saves a few lines of code, and it's easier to read to see the overall layout of the parser.